### DISCUSSION TITLE
Release 3.5.1

### DISCUSSION THREAD

**Discussion by patrickdemooij9** [patrickdemooij9/SeoToolkit.Umbraco#280] [General]
Small update to fix some issues within the package. Support for Umbraco 14 is being worked on and should be out shortly!

## What's Changed
* Minor spelling fix in the RedirectsService exceptions by @erwindamsma in https://github.com/patrickdemooij9/SeoToolkit.Umbraco/pull/272
* Fix regex redirect with custom domain by @luuksommers in https://github.com/patrickdemooij9/SeoToolkit.Umbraco/pull/268
* Add a check to ensure only the content is saved in the current editor by @josephheqj in https://github.com/patrickdemooij9/SeoToolkit.Umbraco/pull/274
* Fix issue with the redirect search (https://github.com/patrickdemooij9/SeoToolkit.Umbraco/issues/270)
* Fix issue with special characters in redirect (https://github.com/patrickdemooij9/SeoToolkit.Umbraco/issues/279)

## New Contributors
* @erwindamsma made their first contribution in https://github.com/patrickdemooij9/SeoToolkit.Umbraco/pull/272
* @luuksommers made their first contribution in https://github.com/patrickdemooij9/SeoToolkit.Umbraco/pull/268
* @josephheqj made their first contribution in https://github.com/patrickdemooij9/SeoToolkit.Umbraco/pull/274

**Full Changelog**: https://github.com/patrickdemooij9/SeoToolkit.Umbraco/compare/3.5.0...3.5.1

<hr /><em>This discussion was created from the release <a href='https://github.com/patrickdemooij9/SeoToolkit.Umbraco/releases/tag/3.5.1'>Release 3.5.1</a>.</em>

**Comment by paulbrown79** [comment]
Great work as ever, looking forward to v14 support 👏

**Comment by ssplmanishyogi** [comment]
Hey @patrickdemooij9 Can you tell me when we will get a new release that will be compatible with Umbraco 14?
